lib/engine/part/path.rb

```diff
@@ -5,18 +5,43 @@
 module Engine
   module Part
     class Path < Base
-      attr_reader :a, :b, :branches, :city, :edges, :junction, :lanes, :lane_index,
+      attr_reader :a, :ab_lanes, :b, :branches, :city, :edges, :exit_lanes, :junction,
```

Comment:
why don't we just call it lanes? 

why is an ab_lane and what's a lane?

Reply:
OK, I can rename.